src/main/java/dev/tourmi/svmm/commands/ForceCommand.java

```diff
@@ -28,7 +29,10 @@ public int defaultExecute(CommandContext<CommandSourceStack> cc) {
         ClientConfig cfg = CommandUtils.getSourceConfig(cc);
         status.forceNext = !status.forceNext;
         String message = status.forceNext ? cfg.TRIGGER_WHEN.get().formatConditionText("Next block mined{0} will be vein mined", " {0}") : "Cancelled vein mine";
-        player.sendSystemMessage(Component.literal(message));
+
+		ServerPlayer serverPlayer = (ServerPlayer) player; 
```

Comment:
Instead of casting `Player` to `ServerPlayer` and `Level` to `ServerLevel`, function signatures should be updated to accept these types directly.